src/inject/index.ts

```diff
@@ -11,14 +11,14 @@ function onMessage({type, data}) {
         case 'add-static-theme': {
             const css = data;
             removeDynamicTheme();
-            createOrUpdateStyle(css);
+            createOrUpdateStyle(css, 'static');
```

Comment:
`case 'add-css-filter':` should use `'filter'` too, so you will have to split the condition.